runtime-light/core/std/containers.h

```diff
@@ -0,0 +1,51 @@
+//  Compiler for PHP (aka KPHP)
+//  Copyright (c) 2024 LLC «V Kontakte»
+//  Distributed under the GPL v3 License, see LICENSE.notice.txt
+
+#pragma once
+
+#include <deque>
+#include <functional>
+#include <list>
+#include <map>
+#include <queue>
+#include <string>
+#include <unordered_map>
+#include <unordered_set>
+#include <utility>
+#include <vector>
+
+namespace kphp {
+
+namespace stl {
```

Comment:
Confusing name `stl`. 

1) Do we actually need extra namespace `stl`? `kphp::string` or `kphp::vector` is not enough?
2) Use `containers` or `types` instead?

Reply:
`stl` here stands for `standard template library`. Moreover, it's the same naming as we use in `memory_resource::stl`. I think it's fine to keep it as it's in order to minimize changes in comparison to our existing functionality.